crates/router/src/core/utils.rs

```diff
@@ -323,6 +333,10 @@ pub async fn construct_submit_evidence_router_data<'a>(
         key_store,
     )
     .await?;
+    let test_mode: Option<bool> = match &merchant_connector_account {
```

Comment:
Move this method in MerchantConnectorAccountType implementation and reuse where ever required